### ISSUE TITLE
Documentation for --config option and config file outside doc root

### ISSUE THREAD

**Issue by JBlocks** [sabbelasichon/typo3-rector#2029]
There is no example or info available how to handle paths if the `--config` option is defined. It is difficult to find out which entry point is used for the rules. The "__DIR__" definition no longer seems to apply here, if the rector.php file is not in the document root dir.

We need examples for this cases:
Rector config: `Build/Apps/Rector/rector.php`
CLI Cmd: `./vendor/bin/typo3-rector process --config Build/Apps/Rector/rector.php`

How to configure the part of `Option::SKIP` here? The `__DIR__ . '/**/TCA/*',` will not work anymore.
```
 $parameters->set(
        Option::SKIP,
        [
            NameImportingPostRector::class => [
              __DIR__ . '/**/TCA/*',
```

Same for if you want to use predefined paths.
```
     // paths to refactor; solid alternative to CLI arguments
    $parameters->set(
        Option::PATHS,
        [
            __DIR__ . '/**/acme_demo/',
            __DIR__ . '/**/acme_rectorMe/',
        ]
    );
```

**Comment by sabbelasichon** [comment]
@JBlocks Thanks. This is true. Actually i have never used the --config parameter. Would be awesome if you could provide some examples.  I thinks what you are looking for is the getcwd() function call instead of __DIR__. So if you could enhance the documentation or add something to the template this would be awesome. The template file can be found here: https://github.com/sabbelasichon/typo3-rector/blob/master/templates/rector.php.dist

**Comment by sabbelasichon** [comment]
Closed by #2033
